vault/blog.community.2021.new-user-survey.md

```diff
@@ -0,0 +1,195 @@
+- Users are now prompted when running [`Delete Node`](https://wiki.dendron.so/notes/eea2b078-1acc-4071-a14e-18299fc28f47/#delete-node), and shown what links will break throughout documentation _before_ confirming deletion.
+  - [`Dendron Doctor`](https://wiki.dendron.so/notes/ZeC74FYVECsf9bpyngVMU/) has also had similar improvements, such as [`findBrokenLinks`](https://wiki.dendron.so/notes/ZeC74FYVECsf9bpyngVMU/#findbrokenlinks), to make it easier to see where documentation improvements can be made.
+
+Many of our new users are coming to Dendron as their first tool for personal knowledge management. For users coming in with prior experience, there was no single tool that folks were migrating from. Instead, we had a massive list of previous tools. For those coming from existing tools, we have a [migration guide](https://wiki.dendron.so/notes/f9b4fc21-7613-4c8a-9257-cec4c06b67f9/) that continues to grow.
+
+> We are looking to the community to [help us improve and expand the source documentation](https://github.com/dendronhq/dendron-site/blob/master/vault/dendron.guides.migration.md)!
+
+### Interoperability
+
+Within the Dendron Discord, users have mentioned their interest in _interoperability_: they want a single source that they export/import their notes to/from. Different tools co-exist in workflows and can be used concurrently.
+
+Our users continue to integrate information managed by Dendron with **pods** or their own tooling. In Dendron, [pods](https://wiki.dendron.so/notes/66727a39-d0a7-449b-a10d-f6c438185d7f/) refer to any tool or location that contains information and how Dendron manages that information: [Markdown files](https://wiki.dendron.so/notes/13c4a608-0a32-4c62-9c7e-2b7f9f2d18bf/), [GitHub issues](https://wiki.dendron.so/notes/PuDUEyUAPmvpBvaFWHDOS/), [Notion](https://wiki.dendron.so/notes/eOw3TC9KgdbBvhyU7pyGl/), and social media (Twitter, Facebook, etc.) are pod examples.
+
+Internally, the Dendron team makes extensive use of pods. For example, all tasks are [exported with the Airtable Pod](https://wiki.dendron.so/notes/c8ljkYY3eVkkmTNumc0C7/) so we can utilize alternative views such as Kanban and better task querying (eg. all bugs that have had no updates in the last 7 days).
+
+### Publishing
+
+Dendron stands out compared to other SSGs due to our UX/DX in _creating and managing_ notes, in addition to the actual _publishing_ of notes. For the 75% of users curious about or interested in publishing: we maintain GitHub Templates for easier, more automated publishing on popular platforms:
```

Comment:
what is SSG (I don't think we define it). We also don't talk about it. since this is a takeaway, we should start with an insight. what have we learned from users about publishing? from that, how does dendron publishing help them?